plugins/autodim.koplugin/main.lua

```diff
@@ -222,9 +234,12 @@ function AutoDim:clearEventHandlers()
     self.onSuspend = nil
 end
 
+--- Reschedule AutoDim task.
+--
+-- This might be happening through AutoWarmth during a ramp down.
+-- Set original intensity, but don't turn fl on actually.
+-- @eventHandler onFrontlightTurnedOff
 function AutoDim:onFrontlightTurnedOff()
```

Comment:
I don't remember what we settled on :/
Will we have to add such `@eventhandler onXYZ` above each such method, like you did here?
But you didn't in the chunk below AutoStandby:onAllowStandby() - and it seems if it was really need this PR would have much more of them :) So, was it from some early idea and we don't need them?